cmake_uninstall.cmake.in

```diff
@@ -0,0 +1,40 @@
+if(NOT EXISTS "@CMAKE_BINARY_DIR@/install_manifest.txt")
+	message(FATAL_ERROR "Cannot find install manifest: @CMAKE_BINARY_DIR@/install_manifest.txt")
+endif()
+
+file(READ "@CMAKE_BINARY_DIR@/install_manifest.txt" files)
+string(REGEX REPLACE "\n" ";" files "${files}")
+foreach(file ${files})
+	message(STATUS "Uninstalling $ENV{DESTDIR}${file}")
+	if(IS_SYMLINK "$ENV{DESTDIR}${file}" OR EXISTS "$ENV{DESTDIR}${file}")
+		if(${CMAKE_VERSION} VERSION_LESS "3.17")
+			exec_program(
+				"${CMAKE_COMMAND}" ARGS "-E remove \"$ENV{DESTDIR}${file}\""
+				OUTPUT_VARIABLE rm_out
+				RETURN_VALUE rm_retval
+				)
+		else()
+			exec_program(
+				"${CMAKE_COMMAND}" ARGS "-E rm -r \"$ENV{DESTDIR}${file}\""
+				OUTPUT_VARIABLE rm_out
+				RETURN_VALUE rm_retval
+				)
+		endif()
+		if(NOT "${rm_retval}" STREQUAL 0)
+			message(FATAL_ERROR "Problem when removing $ENV{DESTDIR}${file}")
+		endif()
+	else(IS_SYMLINK "$ENV{DESTDIR}${file}" OR EXISTS "$ENV{DESTDIR}${file}")
+		message(STATUS "File $ENV{DESTDIR}${file} does not exist.")
+	endif()
+endforeach()
+
+set(OSC_LIB_DIR $ENV{DESTDIR}/@CMAKE_INSTALL_PREFIX@/lib/osc)
+set(OSC_SHARE_DIR $ENV{DESTDIR}/@CMAKE_INSTALL_PREFIX@/share/osc)
```

Comment:
Should be `$ENV{DESTDIR}/@CMAKE_INSTALL_PREFIX@` without a `/` in-between (since it is already in the prefix)

Reply:
fixed